agenta-cli/agenta/__init__.py

```diff
@@ -16,7 +16,6 @@
 
 from .sdk.tracing.llm_tracing import Tracing
 from .sdk.decorators.tracing import instrument
-from .sdk.tracing.callbacks import AgentaLiteLLMHandler
```

Comment:
With respect to this [comment](https://github.com/Agenta-AI/agenta/pull/1726/files#r1622507484), wouldn't it be better to update the import to this?

from .tracing import callbacks

Reply:
yes, thanks!